webapp/portlet/src/main/webapp/vue-apps/hamburger-menu/components/ExoHamburgerMenuNavigation.vue

```diff
@@ -148,23 +176,30 @@ export default {
         }
       });
     },
-    openSecondLevel(contentDetail) {
+    openSecondLevel(contentDetail,value) {
       if (!contentDetail.secondLevel || !this.vueChildInstances[contentDetail.id]) {
         return;
       }
       this.secondLevel = true;
+      this.thirdLevel = false;
       document.dispatchEvent(new CustomEvent('second-level-opened', {
         detail: {'contentDetail': contentDetail}
       }));
 
-      if (this.openedSecondLevel !== contentDetail.id && this.vueChildInstances[contentDetail.id].mountSecondLevel) {
+      if (this.openedSecondLevel !== contentDetail.id && this.vueChildInstances[contentDetail.id].mountSecondLevel || value) {
```

Comment:
this value is always false

Reply:
not it's true from the recent space portlet.